bertin.md

```diff
@@ -0,0 +1,501 @@
+| PAWS-X      | F1       |    0.9035 |    0.9000 | 0.8915 | 0.9020 | 0.8820 |
+| XNLI        | Accuracy |    0.8016 |       WIP | 0.8130 | 0.7876 |    WIP |
+
+</figure>
+
+All of our models attained good accuracy values during training in the masked-language model task —in the range of 0.65— as can be seen in Table 2:
+
+<figure>
+
+<caption>Table 2. Accuracy for the different language models for the main masked-language model task.</caption>
+
+| Model                                              | Accuracy |
+|----------------------------------------------------|----------|
+| [`bertin-project/bertin-roberta-base-spanish (beta)`](https://huggingface.co/bertin-project/bertin-roberta-base-spanish)         | 0.6547   |
+| [`bertin-project/bertin-base-random`](https://huggingface.co/bertin-project/bertin-base-random)                  | 0.6520   |
+| [`bertin-project/bertin-base-stepwise`](https://huggingface.co/bertin-project/bertin-base-stepwise)                | 0.6487   |
+| [`bertin-project/bertin-base-gaussian`](https://huggingface.co/bertin-project/bertin-base-gaussian)                | 0.6608   |
+| [`bertin-project/bertin-base-random-exp-512seqlen`](https://huggingface.co/bertin-project/bertin-base-random-exp-512seqlen)    | 0.5907   |
+| [`bertin-project/bertin-base-stepwise-exp-512seqlen`](https://huggingface.co/bertin-project/bertin-base-stepwise-exp-512seqlen)  | 0.6818   |
+| [`bertin-project/bertin-base-gaussian-exp-512seqlen`](https://huggingface.co/bertin-project/bertin-base-gaussian-exp-512seqlen)  | **0.6873**   |
+
+</figure>
+
+### Downstream Tasks
+
+For simplicity, we will abbreviate the different models as follows:
+
+- **mBERT**: [`bert-base-multilingual-cased`](https://huggingface.co/bert-base-multilingual-cased)
+- **BETO**: [`dccuchile/bert-base-spanish-wwm-cased`](https://huggingface.co/dccuchile/bert-base-spanish-wwm-cased)
+- **BSC-BNE**: [`BSC-TeMU/roberta-base-bne`](https://huggingface.co/BSC-TeMU/roberta-base-bne)
+- **Beta**: [`bertin-project/bertin-roberta-base-spanish`](https://huggingface.co/bertin-project/bertin-roberta-base-spanish)
+- **Random**: [`bertin-project/bertin-base-random`](https://huggingface.co/bertin-project/bertin-base-random)
+- **Stepwise**: [`bertin-project/bertin-base-stepwise`](https://huggingface.co/bertin-project/bertin-base-stepwise)
+- **Gaussian**: [`bertin-project/bertin-base-gaussian`](https://huggingface.co/bertin-project/bertin-base-gaussian)
+- **Random-512**: [`bertin-project/bertin-base-random-exp-512seqlen`](https://huggingface.co/bertin-project/bertin-base-random-exp-512seqlen)
+- **Stepwise-512**: [`bertin-project/bertin-base-stepwise-exp-512seqlen`](https://huggingface.co/bertin-project/bertin-base-stepwise-exp-512seqlen)
+- **Gaussian-512**: [`bertin-project/bertin-base-gaussian-exp-512seqlen`](https://huggingface.co/bertin-project/bertin-base-gaussian-exp-512seqlen)
+
+<figure>
+
+All models were fin-tuned on a single Tesla K80 GPU.
```

Comment:
```suggestion
All models were fine-tuned on a single Tesla K80 GPU.
```
I think this is not correct, but can't recall. We might've used K80 for the first round of experiments. I used A100 for the final round of experiments.